contracts/cw20-stake-external-rewards/src/contract.rs

```diff
@@ -243,6 +317,11 @@ pub fn execute_claim(
     env: Env,
     info: MessageInfo,
 ) -> Result<Response<Empty>, ContractError> {
+    let config = CONFIG.load(deps.storage)?;
+    if config.is_paused {
```

Comment:
What is the intention of this pause state? It just prevents users from claiming rewards?

Reply:
It was meant to disable the claims to get time to investigate things